addons/mail/static/src/models/user_setting/user_setting.js

```diff
@@ -17,13 +17,16 @@ function factory(dependencies) {
             const res = super._created(...arguments);
             this._timeoutIds = {};
             this._loadLocalSettings();
+            this._onStorage = this._onStorage.bind(this);
+            browser.addEventListener('storage', this._onStorage);
```

Comment:
reminder: use `this.messaging.browser` during forward-port (I think also starting from v15.2)